sockeye/utils.py

```diff
@@ -214,6 +215,36 @@ def update(self, labels, preds):
             self.num_inst += n
 
 
+class OnlineMeanAndVariance:
+    def __init__(self):
```

Comment:
probably needs a '-> None' to make the code checkers happy

Reply:
utils is not in typechecked-files yet, but it still makes sense to a proper typing here.